extensions/wikia/Captcha/CaptchaController.class.php

```diff
@@ -5,18 +5,12 @@
  */
 class CaptchaController extends WikiaController {
 
-	const DEFAULT_TEMPLATE_ENGINE = WikiaResponse::TEMPLATE_ENGINE_MUSTACHE;
-
 	/**
 	 * Displays a captcha image. This is used exclusively by FancyCaptcha.
 	 */
 	public function showImage() {
 		$fancyCaptcha = new \Captcha\Module\FancyCaptcha();
-		if ( !$fancyCaptcha->showImage() ) {
-			$this->response->setData( [
-				'error' => wfMessage( 'captcha-no-image' )->escaped(),
-			] );
-		}
+		$fancyCaptcha->showImage();
```

Comment:
We don't need to put additional error message. It's already handled: https://github.com/Wikia/app/blob/dev/extensions/wikia/Captcha/Module/FancyCaptcha.class.php#L260